### DISCUSSION TITLE
posts/how-to-use-idm-in-linux-with-extension/

### DISCUSSION THREAD

**Discussion by giscus** [tazihad/tazihad.github.io#3] [Announcements]
# posts/how-to-use-idm-in-linux-with-extension/

Prerequisite: Flatpak.

https://zihad.com.bd/posts/how-to-use-idm-in-linux-with-extension/

**Comment by ferbulous** [comment]
Hi Zihad, I was wondering if a similar method can be used with Ant Download Manager?
This seems to be better than IDM since it can download encrypted stream

**Comment by enamul6m** [comment]
tried with steamdeck but no luck, idk what i did wrong but in chrome thr idm extention says error and download dont bring the idm window

**Reply by tazihad** [reply]
You are right. Extension doesn't support Chrome anymore. Google has removed the extension from webstore. Probably because it uses Manifest v2.
Only works on firefox.

I have updated the guide with screenshot.

**Reply by Iit01b** [reply]
Zihad, any thoughts about making a chromium-extention pakage?

**Reply by tazihad** [reply]
This is late. But I have updated the guide with another addon that works. Tested with Google Chrome and Firefox.

**Comment by francis0915** [comment]
is still working ? because in external extension its hard to past in One Last Step the node.js even i install the ./install sh the One Last Step still appear in Firefox by the way im using steam os

**Reply by tazihad** [reply]
It is still working. Setting it up is a pain in the a..... But still works if all the locations are correct.

**Comment by shahriar-ahmed-seam** [comment]
Can we use it in Chrome?

**Reply by tazihad** [reply]
Yes, the same extension is available in Chrome.

ps. I have been developing this new Download manager similar to IDM with extension support and youtube download support. Check it out: https://zihad.com.bd/bengal-download-manager/
